pom.xml

```diff
@@ -21,8 +21,9 @@
     </licenses>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <workflow.version>1.14</workflow.version>
+        <jenkins.version>1.625.3</jenkins.version>
+        <java.level>7</java.level>
```

Comment:
@jglick I personally prefer putting it as it makes easier to distinguish it with a quick sight. But if you prefer not having it, I am not against removing it either.